gen/definitions/ntp_server.yaml

```diff
@@ -0,0 +1,54 @@
+---
+name: NTP Server
+class_name: datetimeNtpProvider
+dn: sys/time/prov-[%s]
+ds_description: This data source can read an ntp server or peer.
+res_description: This resource can manage an ntp server or peer.
+doc_path: System/top:NTP/
+doc_category: NTP
```

Comment:
`doc_category` refers to the groups shown in the Terraform registry documentation (folders on the left hand side here: https://registry.terraform.io/providers/netascode/nxos/latest/docs/resources/system). I would suggest to add this resource and data source to the `System` category for now.